distreqx/bijectors/triangular_linear.py

```diff
@@ -0,0 +1,105 @@
+"""Triangular linear bijector."""
+
+from typing import Tuple
+
+import jax
+import jax.numpy as jnp
+from jaxtyping import Array
+
+from ._bijector import AbstractBijector
+from ._linear import AbstractLinearBijector
+
+
+def _triangular_logdet(matrix: Array) -> Array:
+    """Computes the log absolute determinant of a triangular matrix."""
+    return jnp.sum(jnp.log(jnp.abs(jnp.diag(matrix))))
+
+
+class TriangularLinear(AbstractLinearBijector):
+    """A linear bijector whose weight matrix is triangular.
+
+    The bijector is defined as `f(x) = Ax` where `A` is a DxD triangular matrix.
+
+    The Jacobian determinant can be computed in O(D) as follows:
+
+    log|det J(x)| = log|det A| = sum(log|diag(A)|)
+
+    The inverse is computed in O(D^2) by solving the triangular system `Ax = y`.
+
+    The bijector is invertible if and only if all diagonal elements of `A` are
+    non-zero. It is the responsibility of the user to make sure that this is the
+    case; the class will make no attempt to verify that the bijector is
+    invertible.
+    """
+
+    _matrix: Array
+    _is_lower: bool
+
+    def __init__(self, matrix: Array, is_lower: bool = True):
+        """Initializes a `TriangularLinear` bijector.
+
+        Args:
```

Comment:
Can you have the comment style match this: https://github.com/lockwo/distreqx/blob/main/distreqx/bijectors/diag_linear.py#L36. This just makes it so the documentation rendering works in a certain way (and is consistent)